CorsixTH/Lua/epidemic.lua

```diff
@@ -234,25 +239,22 @@ end
  the length of the timer. If an infected patient leaves the
  hospital it's discovered instantly and will result in a fail.]]
 function Epidemic:checkNoInfectedPlayerHasLeft()
-  local function infected_patient_left()
-    for _, infected_patient in ipairs(self.infected_patients) do
-      local px, py = infected_patient.tile_x, infected_patient.tile_y
-      -- If leaving and longer in the hospital
-      if infected_patient.going_home
-          and not infected_patient.cured
-          and px and py and not self.hospital:isInHospital(px,py) then
-        return true
-      end
-    end
-    return false
-  end
+  if self.result_determined then return end
 
-  if infected_patient_left() and not self.result_determined then
-    self.result_determined = true
-    if not self.inspector then
-      self:spawnInspector()
+  -- Check whether a patient has left.
+  for _, infected_patient in ipairs(self.infected_patients) do
+    local px, py = infected_patient.tile_x, infected_patient.tile_y
+    -- If leaving and longer in the hospital
```

Comment:
This is a typo in my original one should read "-- If leaving and **no ** longer in the hospital"